Rules/ReviewUnusedParameter.cs

```diff
@@ -21,8 +21,60 @@ namespace Microsoft.Windows.PowerShell.ScriptAnalyzer.BuiltinRules
 #endif
     public class ReviewUnusedParameter : IScriptRule
     {
+        private readonly string TraverseArgName = "traverseList";

```

Comment:
This parameter needs to be documented here: https://github.com/PowerShell/PSScriptAnalyzer/blob/master/docs/Rules/ReviewUnusedParameter.md
Also, naming it something like `CommandsToTraverse` would be better.

Reply:
makes sense, has been updated and docs been added :)